### PR TITLE
Add payout page

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2068** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2068" target="_blank">Graphite</a> 
    * **PR #2071** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2071" target="_blank">Graphite</a> 
      * **PR #2072** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2072" target="_blank">Graphite</a> 
        * **PR #2073** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2073" target="_blank">Graphite</a> 
          * **PR #2074** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2074" target="_blank">Graphite</a> 
            * **PR #2075** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2075" target="_blank">Graphite</a> 
              * **PR #2076** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2076" target="_blank">Graphite</a> 
                * **PR #2077** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2077" target="_blank">Graphite</a> 
                  * **PR #2078** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2078" target="_blank">Graphite</a> 
                    * **PR #2079** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2079" target="_blank">Graphite</a> 
                      * **PR #2080** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2080" target="_blank">Graphite</a> 
                        * **PR #2082** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2082" target="_blank">Graphite</a> 
                          * **PR #2084** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2084" target="_blank">Graphite</a> 
                            * **PR #2085** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2085" target="_blank">Graphite</a> 
                              * **PR #2086** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2086" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2086?utm_source=stack-comment).

File: src/components/Create/components/pages/Payouts/PayoutsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      initialValues={initialValues}
+      name="fundingTarget"
+      colon={false}
+      layout="vertical"
+      onFinish={goToNextPage}
+      scrollToFirstError
+    >
+      <Space direction="vertical" size="large" style={{ width: '100%' }}>
+        <RecallCard show={['fundingCycles', 'fundingTarget']} />
+        <h2>How would you like to distribute payments?</h2>
```
Comment (tomquirk) [inline src/components/Create/components/pages/Payouts/PayoutsPage.tsx:32]:
```suggestion
        <h2><Trans>How would you like to distribute payments?</Trans></h2>
```

File: src/components/Create/components/pages/Payouts/PayoutsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                  in the next step.
+                </Trans>
+              }
+            />
+          </Selection>
+        </Form.Item>
+        {selection && (
+          <Space direction="vertical" style={{ width: '100%' }}>
+            <h2>Who's getting paid?</h2>
+            <p>Add wallet addresses or juicebox projects to receive payouts.</p>
```
Comment (tomquirk) [inline src/components/Create/components/pages/Payouts/PayoutsPage.tsx:62]:
```suggestion
            <p>Add wallet addresses or Juicebox projects to receive payouts.</p>
```

+ trans these boys

File: src/components/Create/components/pages/Payouts/components/PayoutsList/PayoutsList.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import { t } from '@lingui/macro'
```
Comment (tomquirk) [inline src/components/Create/components/pages/Payouts/components/PayoutsList/PayoutsList.tsx:1]: Fantastic job with this component <3

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2086?utm_source=stack-comment).


Job Reference: TLpS7mRK611WyBbtlMOK

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2086?utm_source=stack-comment).


Job Reference: 7mQp0aCulfn1WjwNMx70

**Pull request merged**